codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -273,7 +272,7 @@ public final void finishHandshake(Channel channel, FullHttpResponse response) {
             // Delay the removal of the decoder so the user can setup the pipeline if needed to handle
             // WebSocketFrame messages.
             // See https://github.com/netty/netty/issues/4533
-            channel.eventLoop().execute(new OneTimeTask() {
+            channel.eventLoop().execute(new Runnable() {
```

Comment:
@johnou just a hack to save on object allocation. The `OneTimeTask` is both a `Runnable` and a `LinkedQueueNode`, so you don't need an extra `LinkedQueueNode` when adding it to the queue.